tests/test_database.py

```diff
@@ -384,6 +384,26 @@ def test_dbc_load_empty_choice(self):
         self.assertEqual(message.signals[2].name, 'no_choice')
         self.assertEqual(message.signals[2].choices, None)
 
```

Comment:
is there no test for changing the order of the signals? (would it make sense to sort the messages as well, BTW?)

Reply:
I think dbc retains the order of messages chronologically.

Reply:
I'm not aware of any currently existing tests for the sorting of signals which happens by default. I have added a test for *not* sorting the signals in commit fc68778c9e3ec08ac6520b648f12e05f63e7babf, though. But I intend to add a test for passing a function to the new `sort_signals` parameter and also for leaving it out.

Reply:
I don't think it makes much sense to sort the messages. If a user wants them sorted they can easily add a `db.messages.sort(key=lambda msg: msg.frame_id)` to their script.

Reply:
good point